packages/edit-site/src/components/page-templates/fields.js

```diff
@@ -156,13 +156,13 @@ export const activeField = {
 	getValue: ( { item } ) => item._isActive,
 	render: function Render( { item } ) {
 		const activeLabel = item._isCustom
-			? __( 'Active when used' )
-			: __( 'Active' );
+			? _x( 'Active when used', 'template' )
```

Comment:
Beyond the scope of this PR, but I wonder if this one could use a tooltip for the user. It might also be a little confusing for translators.

For both audiences, it could read as tautological.